example/gluon/DCgan/README.md

```diff
@@ -0,0 +1,31 @@
+# DCgan in MXNet
+
+train the dcgan with mxnet, and eval dcgan model with inception_score.
```

Comment:
[Deep Convolutional Generative Adversarial Network (DCGAN)](https://arxiv.org/abs/1511.06434) implementation with Apache MXNet GLUON. This implementation uses, [inception_score](https://github.com/openai/improved-gan) to evaluate the model.

You can use this reference implementation on MNIST and CIFAR10 dataset.